dts/bindings/vendor-prefixes.txt

```diff
@@ -1,19 +1,22 @@
-Device tree binding vendor prefix registry. Keep this list in
-alphabetical order.
-
-This isn't an exhaustive list, but you should add new prefixes to it
-before using them to avoid name-space collisions.
-
-The contents of this file are parsed during documentation generation.
-Anything above the following line of '-' characters is ignored. Every
-non-empty line after that should contain a line in this format:
-
-<vendor-prefix><TAB><Full name of vendor>
-
---------------------------------------------------
+# Device tree binding vendor prefix registry. Keep this list in
+# alphabetical order.
+#
+# This isn't an exhaustive list, but you should add new prefixes to it
+# before using them to avoid name-space collisions.
+#
+# The contents of this file are parsed during documentation generation.
+# Anything that starts with a '#' is treated as a comment and ignored.
+# Non-empty lines should be in this format:
+#
+# <vendor-prefix><TAB><Full name of vendor>
 
+70mai	70mai Co., Ltd.
```

Comment:
That's a good point. I can only find one usage in Linux: https://github.com/torvalds/linux/blob/master/arch/arm/boot/dts/mstar-mercury5-ssc8336n-midrived08.dts

I guess that dt-schema would barf on this, which is weird. I'll just drop it from this PR since we don't need it. Thanks.